[TA] Allow only one actions per actionType, otherwise, throw IllegalArgumentException (#22723)
Azure/azure-sdk-for-java: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

  `AnalyzeActionsResultPagedIterable`, `AnalyzeHealthcareEntitiesPagedFlux`, and `AnalyzeHealthcareEntitiesPagedIterable`.
  
### Breaking Changes

> - Changed behavior in `beginAnalyzeActions` API where now accepts a single action per action type. 

Reviewer: ```suggestion
- Changed behavior in `beginAnalyzeActions` API where now accepts up to one action only per action type. 
```

>   An `IllegalArgumentException` is raised if duplicate actions are passed.

Reviewer: ```suggestion
  An `IllegalArgumentException` is raised if multiple actions of the same type are passed.
```
